src/cpp/src/icontinuous_batching.hpp

```diff
@@ -59,6 +60,7 @@ class ContinuousBatchingPipeline::IContinuousBatchingPipeline {
     std::mutex m_inputs_embedder_mutex;
 
     void stream_tokens(const std::shared_ptr<ThreadedStreamerWrapper>& streamer_ptr, const GenerationHandle& handle);
+    std::string add_image_tags_to_prompt(const std::string& prompt, const std::vector<ov::Tensor>& rgbs);
```

Comment:
it can be a free function, defined in unnamed namespace

Reply:
Done.